src/components/BookCard/BookCard.css

```diff
@@ -0,0 +1,47 @@
+.card {
+  width: 180px;
+  height: 290px;
+  border-color: black;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  background-color: antiquewhite;
+  margin-bottom: 15px;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 200;
+  color: black;
+  border: 2px solid rgb(185, 185, 185);
+}
+.pic {
+  height: 150px;
+  background-color: rgb(174, 112, 68);
+  border-radius: 3px;
+  margin-top: 0px;
+}
+h3 {
+  margin-top: 3px;
+  text-align: center;
+  font-size: medium;
+}
+.btn {
+  width: 60%;
+
+  height: 30px;
+  font-weight: 600;
+  background-color: rgb(255, 238, 1);
```

Comment:
Only use a single type of code codes, either a hexadecimal representation or a rgb representation throughout the application